### PR TITLE
Laboratory work #4, Gellertova Loreleya - 23FPL1

### PR DESCRIPTION
**Pull request by LoraGellertova**

### CODE REVIEW

File: lab_4_retrieval_w_clustering/main.py [hunk 1]
```diff
@@ -4,9 +4,9 @@
 Vector search with clusterization
 """
 
-# pylint: disable=undefined-variable, too-few-public-methods, unused-argument, duplicate-code, unused-private-member, super-init-not-called
```
Comment (EIzharikov) [inline lab_4_retrieval_w_clustering/main.py:7]: revert these changes

File: lab_4_retrieval_w_clustering/main.py [hunk 2]
```diff
@@ -40,6 +44,9 @@ def __init__(self) -> None:
         """
         Initialize an instance of the BM25Vectorizer class.
         """
+        self._corpus = []
+        super().__init__(self._corpus)
```
Comment (EIzharikov) [inline lab_4_retrieval_w_clustering/main.py]: super init firstly

File: lab_4_retrieval_w_clustering/main.py [hunk 3]
```diff
@@ -113,6 +155,19 @@ def put_corpus(self, corpus: Corpus) -> None:
                 or if input arguments are empty,
                 or if methods used return None.
         """
+        if not isinstance(corpus, list) or len(corpus) == 0:
+            raise ValueError
+
+        for text in corpus:
+            self.__documents = self._tokenizer.tokenize(text) # список токенизированных абзацев
```
Comment (EIzharikov) [inline lab_4_retrieval_w_clustering/main.py]: delete comments

File: lab_4_retrieval_w_clustering/start.py [hunk 4]
```diff
@@ -42,6 +42,8 @@ def open_files() -> tuple[list[str], list[str]]:
             documents.append(file.read())
     with open("assets/stopwords.txt", "r", encoding="utf-8") as file:
         stopwords = file.read().split("\n")
+
+    print("do pr me")
```
Comment (EIzharikov) [inline lab_4_retrieval_w_clustering/start.py]: do start pls

**Review by EIzharikov — COMMENTED** [review]
Fix comments, make CI green and be ready for Friday. Good luck!

File: lab_4_retrieval_w_clustering/main.py (hunk 1 above)
Comment (LoraGellertova) [reply lab_4_retrieval_w_clustering/main.py:7]: done

File: lab_4_retrieval_w_clustering/main.py (hunk 2 above)
Comment (LoraGellertova) [reply lab_4_retrieval_w_clustering/main.py]: done

File: lab_4_retrieval_w_clustering/main.py (hunk 3 above)
Comment (LoraGellertova) [reply lab_4_retrieval_w_clustering/main.py]: done

File: lab_4_retrieval_w_clustering/start.py (hunk 4 above)
Comment (LoraGellertova) [reply lab_4_retrieval_w_clustering/start.py]: i'm trying to reach 8
